stubs/pyaudio/METADATA.toml

```diff
@@ -1,4 +1,7 @@
 version = "0.2.*"
+# There is no web portal for the source, this is the official source link:
+# https://people.csail.mit.edu/hubert/pyaudio/#sources
+upstream_repository = "https://people.csail.mit.edu/hubert/git/pyaudio.git"
```

Comment:
This is a bit weird; neither link would take us to a version of the source that you could view in a web browser. I think I'd prefer just doing this tbh:

```suggestion
# There is no web portal for the source, this is the official source link:
# https://people.csail.mit.edu/hubert/pyaudio/#sources
```